### DISCUSSION TITLE
1.19 Support

### DISCUSSION THREAD

**Discussion by audiodk** [itsmemac/Block-Particles#1] [General]
Can you update this to support 1.19.  I have this version installed and the particles do not work

**Comment by itsmemac** [comment]
Looking into updating soon

**Comment by itsmemac** [comment]
> Can you update this to support 1.19. I have this version installed and the particles do not work

https://github.com/itsmemac/Block-Particles/releases/tag/v1.12.2

**Reply by audiodk** [reply]
Awesome thank you.  Noticed one of the particles isn't working.  I think it is the doublespiral one

**Reply by itsmemac** [reply]
Ya.. working on fixing it
Will fix it in the next release
